packages/apps/explorer/src/context/networks-context.tsx

```diff
@@ -109,34 +119,39 @@ const NetworkContextProvider = (props: {
     };
   }, [storageListener]);
 
-  const setActiveNetworkByKey = (networkId: string): void => {
-    const network = networks.find((x) => x.networkId === networkId)!;
-    setActiveNetwork(network);
+  const setActiveNetworkByKey = (networkSlug: string): void => {
+    const network = networks.find((x) => x.slug === networkSlug);
+
+    if (!network) {
+      addToast({
+        type: 'negative',
+        label: 'Network not found',
+        body: `network ${networkSlug} is depricated`,
```

Comment:
Should we even add a toast when the network isn't found? Wouldn't it be better to just forward to `/mainnet` and ignore the wrong network.

Also, what does it mean tha ta network is deprecated, does it work now, and later not anymore?


Typo
```suggestion
        body: `network ${networkSlug} is deprecated`,
```

Reply:
my english is not so good :)